src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -86,4 +87,31 @@ private static void setCheckName(InputConfiguration.Builder inputConfigBuilder,
         final String checkPath = getPackageFromFilePath(filePath) + "." + checkName + "Check";
         inputConfigBuilder.setCheckName(checkPath);
     }
+
+    private static void setCheckAttributes(InputConfiguration.Builder inputConfigBuilder,
+                                           String filePath, List<String> lines)
+                    throws CheckstyleException {
+        int lineNo = 2;
+        for (String line = lines.get(lineNo); !line.contains("*/"); line = lines.get(++lineNo)) {
+            if (line.isEmpty()) {
+                continue;
+            }
+            try {
+                final int equalPosition = line.indexOf('=');
+                final String attribute = line.substring(0, equalPosition - 1);
```

Comment:
please reuse standard class Property 
https://www.technicalkeeda.com/java-tutorials/java-properties-load-reader
we do not care about performance right now at all !!! 
We need support of line wrap for values - https://stackoverflow.com/a/5576197/1015848 .

Please in this PR update 
https://github.com/checkstyle/checkstyle/blob/15659661b43a5caaed21b789d767ddd0c07f8f26/src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationonsameline/InputAnnotationOnSameLineCheck3.java#L2-L3
to wrap a line and make Input parse-able.

We need to remove 'token' from linelenght allowance
https://github.com/checkstyle/checkstyle/commit/15659661b43a5caaed21b789d767ddd0c07f8f26#diff-1e338c0c9df28dfa0d2a973b110fd6b2aaa68f8d954f401abb798cfb8ad509daR22

Reply:
> please reuse standard class Property

@romani done, please see

Reply:
> We need support of line wrap for values

moved to https://github.com/checkstyle/checkstyle/issues/10163
multiple configs may need refactoring, suppression needs to ve removed
i'll send PR once it is approved